plugins/channeltx/filesource/filesource.cpp

```diff
@@ -54,8 +54,7 @@ FileSource::FileSource(DeviceAPI *deviceAPI) :
     ChannelAPI(m_channelIdURI, ChannelAPI::StreamSingleSource),
     m_deviceAPI(deviceAPI),
 	m_frequencyOffset(0),
-	m_basebandSampleRate(0),
```

Comment:
Why removing it from the initialization list ?

Reply:
"m_linearGain" is never used.
And the line with 'm_basebandSampleRate' only got comma removed.